st2actions/st2actions/container/base.py

```diff
@@ -63,6 +67,10 @@ def _get_runner(self, runnertype_db, action_db, liveaction_db):
         runner.libs_dir_path = self._get_action_libs_abs_path(action_db.pack,
                                                               action_db.entry_point)
 
+        # For re-run, get the ActionExecutionDB in which the re-run is based on.
+        rerun_ref_id = runner.context.get('re-run', {}).get('ref')
+        runner.rerun_ex_ref = ActionExecution.get(id=rerun_ref_id) if rerun_ref_id else None
```

Comment:
Actually, ignore that comment, sorry.

I see you do it in the constructor bellow. It's not ideal since it's still dynamic and pylint cant detect those variables so it would be better to assign it as class level variable, but I'm fine with doing this later in another PR.